modules/ignite-cli-2/src/test/java/org/apache/ignite/cli/commands/cliconfig/ConfigTest.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.ignite.cli.commands.cliconfig;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import org.apache.ignite.cli.config.Config;
+import org.junit.jupiter.api.Test;
+
+public class ConfigTest {
+    @Test
+    public void testSaveLoadConfig() throws IOException {
+        File tempFile = File.createTempFile("cli", null);
```

Comment:
You can get the TempDir from the Junit infrastructure. Just path the path as a method parameter and annoyed it with TempDir